### ISSUE TITLE
New lines in the WYSIWYG editor not showing up

### ISSUE THREAD

**Issue by miranmirza** [APXEOLOG/nodebb-plugin-bbcodes#6]
When I put in a line break they don't show up when I save the text, I assume it's likely because the finalised text doesn't convert the newline character to <br>. This line and regex ".replace(/\n/g, "<br />");" should address that issue, I'm just not sure where it needs to go.

**Comment by APXEOLOG** [comment]
Yeah, the problem comes when you disable "HTML Sanitization". For example Markdown plugin replace `\n` with `<br>` itself, so i'm not sure how to do it right.
